datasets/financial_phrasebank/financial_phrasebank.py

```diff
@@ -0,0 +1,150 @@
+As a result, sentences which have a sentiment that is not relevant from an
+economic or financial perspective are considered neutral.
+
+This release of the financial phrase bank covers a collection of 4840
+sentences. The selected collection of phrases was annotated by 16 people with
+adequate background knowledge on financial markets. Three of the annotators
+were researchers and the remaining 13 annotators were master’s students at
+Aalto University School of Business with majors primarily in finance,
+accounting, and economics.
+
+Given the large number of overlapping annotations (5 to 8 annotations per
+sentence), there are several ways to define a majority vote based gold
+standard. To provide an objective comparison, we have formed 4 alternative
+reference datasets based on the strength of majority agreement: all annotators
+agree, >=75% of annotators agree, >=66% of annotators agree and >=50% of
+annotators agree.
+"""
+
+_HOMEPAGE = "https://www.kaggle.com/ankurzing/sentiment-analysis-for-financial-news"
+
+_LICENSE = "Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported License"
+
+_URL = "https://www.researchgate.net/profile/Pekka_Malo/publication/251231364_FinancialPhraseBank-v10/data/0c96051eee4fb1d56e000000/FinancialPhraseBank-v10.zip"
+
+
+_VERSION = datasets.Version("1.0.0")
+
+
+class FinancialPhraseBankConfig(datasets.BuilderConfig):
+    """BuilderConfig for FinancialPhraseBank."""
+
+    def __init__(
+        self,
+        split,
+        **kwargs,
+    ):
+        """BuilderConfig for Discovery.
+        Args:
+          filename_bit: `string`, the changing part of the filename.
+        """
+
+        super(FinancialPhraseBankConfig, self).__init__(name=f"sentences_{split}agree", version=_VERSION, **kwargs)
+
+        self.path = os.path.join("FinancialPhraseBank-v1.0", f"Sentences_{split.title()}Agree.txt")
+
+
+class FinancialPhraseBank(datasets.GeneratorBasedBuilder):
+    __doc__
+
+    BUILDER_CONFIGS = [
+        FinancialPhraseBankConfig(
+            split="all",
+            description="Sentences where all annotators agreed",
+        ),
+        FinancialPhraseBankConfig(split="75", description="Sentences where at least 75% of annotators agreed"),
+        FinancialPhraseBankConfig(split="66", description="Sentences where at least 66% of annotators agreed"),
+        FinancialPhraseBankConfig(split="50", description="Sentences where at least 50% of annotators agreed"),
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "sentence": datasets.Value("string"),
+                    "label": datasets.features.ClassLabel(
+                        names=[
+                            "positive",
+                            "neutral",
+                            "negative",
+                        ]
```

Comment:
```suggestion
                        names=[
                            "negative",
                            "neutral",
                            "positive",
                        ]
```
usually we set "negative" at index 0 (by convention)

You'll need to regenerate the dataset_infos.json after this change